.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+## Description
+
+<!-- Explain what this Pull Request changes -->
+
+<!-- Link any issues if applicable.
+Use keywords from https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests
+-->
+
+<!-- Uncomment based on the type of your changes below -->
+
+<!--
```

Comment:
Maybe remove this entire thing since it'd be obvious from the pr description

Reply:
https://github.com/eludris/elusite/pull/2/commits/1233f1fe67d833ea62951ab8e1d959e6fecce00e

Reply:
https://github.com/eludris/elusite/pull/2/commits/1233f1fe67d833ea62951ab8e1d959e6fecce00e